autokey-blueprints/0-setup/main.tf

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  default_suffix         = var.suffix != "" ? var.suffix : random_string.suffix.result
+  autokey_key_project_id = var.create_autokey_key_project ? module.autokey_key_project[0].project_id : var.autokey_key_project_id
+  autokey_folder_id      = var.create_autokey_folder ? google_folder.autokms_folder[0].folder_id : var.autokey_folder_id
+}
+
+resource "random_string" "suffix" {
+  length  = 4
+  special = false
+  upper   = false
+}
+
+resource "google_folder" "autokms_folder" {
```

Comment:
@tdbhacks this should be `autokey_resource_folder`?
as this folder will host resources protected by the `kms keys in kms_project` configured as part of `autokeyConfig` on it. 
It is not necessary that `kms_project` will be inside this folder

Reply:
Name updated! Tks!